styles/AboutPage.css

```diff
@@ -0,0 +1,394 @@
+
+.about_container_header {
+  margin-top: 3rem;
+  padding: 15px 12px 12px 12px;
+  background-image: url("/Assets/backgrounds/background-1.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: var(--light_background);
+}
+
+.medium_text {
+  text-align: center;
+  font-weight: 200;
+  color: var(--primary);
+  font-size: 1.5rem;
+  margin-bottom: 10px;
+}
+
+.about_text2 {
+  color: var(--common);
+  font-weight: bolder;
+  font-size: 7vw;
+  font-family: Inter, sans-serif;
+  text-align: center;
+}
+
+.description_container {
+  background-color: var(--light);
+  padding: 24px 9.5px 24px 9.5px;
+}
+
+.description-note {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: larger;
+  line-height: 25px;
+  text-align: center;
+}
+
+.contact_via_email {
+  background-color: var(--light_background);
+  padding: 29px 44px 29px 44px;
+  text-align: center;
+}
+
+.email_note {
+  line-height: 25px;
+  font-size: 15px;
+  color: var(--light-grey);
+}
+
+.link_to_email {
+  text-decoration: none;
+  color: var(--dark_secondary);
+  font-weight: bold;
+}
+
+.link_to_email:hover {
+  text-decoration: underline;
+  color: var(--dark_secondary);
+}
+
+.logo_container {
+  padding: 30px 60px 30px 60px;
+  border-top: 2px solid var(--light_background);
+  border-bottom: 2px solid var(--light_background);
+  margin-top: 35px;
+}
+
+.event_sample {
+  padding: 34px 20px 34px 20px;
+  border-top: 1px solid var(--secondary);
+  margin-top: 19px;
+}
+
+.sample_header {
+  text-align: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.text5 {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  padding-bottom: 10px;
+}
+
+.text6 {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  padding-top: 10px;
+  color: var(--light-grey);
+}
+
+.event_details {
+  position: absolute;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  background-color: var(--transparent_primary);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0 30px 0 30px;
+  color: var(--light);
+  z-index: 0;
+  visibility: hidden;
+}
+
+.pass_event_cards {
+  margin-top: 35px;
+  position: relative;
+}
+
+.pass_event_cards:hover .event_details {
+  visibility: visible;
+}
+
+.event_image {
+  width: 100%;
+}
+
+.event-date {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+}
+
+.description {
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .about_content {
+    padding: 90px 258px 12px 258px;
```

Comment:
- [OPTIONAL] In some desktop screen sizes, your design in this section gets awkwardly weird due to extensive padding. kindly consider solving this issue by lowering the left-right padding in the following screen size.
![qw](https://user-images.githubusercontent.com/96683843/235316312-bd06ecf9-2c90-42dc-8d72-b0c0b273968c.jpg)